modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator_{context}"]
+= Upgrading the MetalLB Operator
+
+
+.Prerequisites
+
+* Access the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Verify that the `metallb-system` namespace still exists:
++
+[source,terminal]
+----
+$ oc get namespaces | grep metallb-system
+----
++
+.Example output
+[source,terminal]
+----
+metallb-system                                     Active   31m
+----
+
+. Verify the `metallb` custom resource still exists:
++
+[source,terminal]
+----
+$ oc get metallb -n metallb-system
+----
++
+.Example output
+[source,terminal]
+----
+NAME      AGE
+metallb   33m
+----
+
+. Follow the guidance in "Installing from OperatorHub using the CLI" to install the latest {product-version} version of the MetalLB Operator.
++
+[NOTE]
+====
+When installing the latest {product-version} version of the MetalLB Operator, you must install the Operator to the same namespace it was previously installed to. 
+====
+
+. Verify the upgraded version of the Operator is now the {product-version} version.
++
+[source,terminal]
+----
+# oc get csv -n metallb-system
```

Comment:
Does this command need to be run as root on the local installation host? If not, you could replace the `#` prompt with `$`.